webcrypto/tests/wpt-patches/WebCryptoAPI__sign_verify__rsa.js.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/WebCryptoAPI/sign_verify/rsa.js b/WebCryptoAPI/sign_verify/rsa.js
+index 5abadd3d4..c1673065a 100644
+--- a/WebCryptoAPI/sign_verify/rsa.js
++++ b/WebCryptoAPI/sign_verify/rsa.js
+@@ -1,8 +1,8 @@
+ 
+ function run_test() {
+-    setup({explicit_done: true});
++    // setup({explicit_done: true});
+ 
+-    var subtle = self.crypto.subtle; // Change to test prefixed implementations
```

Comment:
I wonder if it won't be easier to actually set `self` to `globalThis` which seems to be what it meant to do https://developer.mozilla.org/en-US/docs/Web/API/Window/self